source/modules/collection-api/address/overview.md.erb

```diff
@@ -12,15 +12,15 @@
 }
 ```
 
-Numerous resources have associated location data: Sale payments have an associated billing address, events occur at a particular address, or an account may have a shipping address associated with it. Inspetor standardizes the format in which we collect this data as follows:
+Numerous resources have associated location data: Order payments have an associated billing address, deliveries have the delivery address, or an user may have an address associated with it. Inspetor standardizes the format in which we collect this data as follows:
```

Comment:
Just to keep the PRs modularized, the change from Sale to Order here could hold until the PR that's doing the Orders endpoint, right?

Reply:
The same regarding the change from events to deliveries

Reply:
I changed already because I will probably forget to change after